### PR TITLE
Fix#118 - auth constraints based on consent attributes

### PR DESCRIPTION
**Pull request by ginglass**
- Created ID2 to security profile
- Added auth constraints based on consent attributes and expected error code

### CODE REVIEW

File: open-banking-brasil-financial-api-1_ID2-ptbr.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
+%%%
+
+    #
+    # Open Banking Brasil Financial-grade API Security Profile 1.0 Implementers Draft 1
+    # (open-banking-brasil-financial-api-1_ID1)
```
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID2-ptbr.md:5]: Draft numbers need to change

File: open-banking-brasil-financial-api-1_ID2-ptbr.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
+%%%
+
+    #
+    # Open Banking Brasil Financial-grade API Security Profile 1.0 Implementers Draft 1
```
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID2-ptbr.md:4]: Draft number needs to change

File: open-banking-brasil-financial-api-1_ID2-ptbr.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+    # (open-banking-brasil-financial-api-1_ID1)
+    #
+    #
+
+    title = "Open Banking Brasil Financial-grade API Security Profile 1.0 Implementers Draft 1"
```
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID2-ptbr.md:9]: Draft is 2

File: open-banking-brasil-financial-api-1_ID2-ptbr.md [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+    abbrev = "OBB-FAPI-1-ID1"
+    ipr = "none"
+    workgroup = "Open Banking Brasil GT Security"
+    keyword = ["FAPI", "Open Banking Brasil GT Security"]
+
+    [seriesInfo]
+    name = "Internet-Draft"
+    status = "standard"
+    value = "open-banking-brasil-financial-api-1_ID1-ptbr"
```
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID2-ptbr.md:18]: File is different.

File: all-documents.md [hunk 5]
```diff
@@ -479,7 +479,10 @@ Além dos requisitos descritos nas disposições de segurança do Open Banking B
 4. deve garantir que os *access tokens* são emitidos com os *scopes* necessários para permitir acesso aos dados especificados em elemento *Permission* do Consentimento (Consent Resource Object) relacionado;8
 5. não deve rejeitar pedido de autorização com *scopes* além do necessário para permitir acesso a dados definidos em elemento *Permission* do Consentimento (Consent Resource Object) relacionado;
 6. pode reduzir o escopo solicitado para um nível que seja suficiente para permitir o acesso aos dados definidos em elemento *Permission* do Consentimento (Consent Resource Object) relacionado;
-7. deve manter registros sobre o histórico dos consentimentos para permitir a adequada formação de trilhas de auditoria em conformidade com a regulação em vigor.
+7. deve manter registros sobre o histórico dos consentimentos para permitir a adequada formação de trilhas de auditoria em conformidade com a regulação em vigor;
```
Comment (RalphBragg) [inline all-documents.md:482]: Can these requirements be added to the security profile ID2.

**Review by RalphBragg — CHANGES_REQUESTED** [review]

File: all-documents.md [hunk 6]
```diff
@@ -479,7 +479,10 @@ Além dos requisitos descritos nas disposições de segurança do Open Banking B
...
+8. deve retornar falha na autenticação e o código de retorno _access_denied_ no parâmetro _erro_ (como especificado na seção 4.1.2.1 da RFC 6749) caso o CPF do usuário autenticado não seja o mesmo indicado no elemento _loggedUser_ do Consentimento (Consent Resource Object);
```
Comment (ediemerson-br) [inline all-documents.md:483]: Change RFC 6749 to [RFC6749] in all documents.

**Review by ediemerson-br — CHANGES_REQUESTED** [review]

File: open-banking-brasil-financial-api-1_ID2.md [hunk 7]
```diff
@@ -0,0 +1,408 @@
...
+# Acknowledgements
+
+With thanks to all who have set the foundations for secure and safe data sharing through the formation of the OpenID Foundation FAPI Working Group, the Open Banking Brasil GT Security and to the pioneers who will stand on their shoulders.
+
+The following people contributed to this document:
+
+* Ralph Bragg (Raidiam)
+* Joseph Heenan (Authlete)
+* Alexandre Siqueira (Mercado Pago)
+* Marcos Rodrigues (Itaú)
```
Comment (RalphBragg) [inline open-banking-brasil-financial-api-1_ID2.md:398]: Add your name @ginglass

**Review by RalphBragg — CHANGES_REQUESTED** [review]
Add your name please to the contributors section of the english specification.

**Pull request merged**